prboom2/src/m_menu.c

```diff
@@ -2959,6 +2961,8 @@ setup_menu_t audiovideo_settings[] = {
   { "Vertical Sync", S_YESNO, m_conf, G_X, dsda_config_render_vsync },
   { "Uncapped Framerate", S_YESNO, m_conf, G_X, dsda_config_uncapped_framerate },
   { "FPS Limit", S_NUM, m_conf, G_X, dsda_config_fps_limit },
+  { "GL Lighting", S_CHOICE, m_conf, G_X, dsda_config_gl_light_mode, 0, gl_light_mode_list },
```

Comment:
Maybe "GL Light Banding" and `gl_light_banding_mode`

Reply:
Light fade mode?  Light drop-off mode?  Light quantization mode?

Reply:
"GL Distant Light Fade"?

Reply:
GL Light Fade sounds good to me